cypress/tests/core/basic/user-group-membership-control-panel.js

```diff
@@ -39,120 +39,124 @@ describe('User Group Membership Control Panel test for NOT many users and many g
   });
   it('Should update group membership for: one user and one group', () => {
     cy.visit('/controlpanel/usergroupmembership');
+    cy.wait(2000);
```

Comment:
It's better to wait for a specific state then a random sleep, this will slow the tests down.